tests/nightly/nightly-kickoff.sh

```diff
@@ -59,51 +54,67 @@ nightlystatus=()
 count=0
 
 for i in $nightly_list_var; do
-    echo "Executing nightly test $i"
-    drone exec --trusted -e test="pybot -d $i --suite $i tests/manual-test-cases/" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
+    echo "Executing nightly test $i vSphere 6.5"
+    drone exec --trusted -e test="pybot -d 65/$i --suite $i tests/manual-test-cases/" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
+
+    if [ $? -eq 0 ]
+    then
+    echo "Passed"
+    nightlystatus[$count]="Pass"
+    else
+    echo "Failed"
+    nightlystatus[$count]="FAIL"
+    fi
+
+    mv *.log 65/$i
+    mv *.zip 65/$i
+    ((count++))
+    echo $count
+done
+
+for i in $nightly_list_var; do
+    echo "Executing nightly test $i on vSphere 6.0"
+    drone exec --trusted -e test="pybot --variable ESX_VERSION:3620759 --variable VC_VERSION:3634791 -d 60/$i --suite $i tests/manual-test-cases/" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
 
     if [ $? -eq 0 ]
     then
     echo "Passed"
-    nightlystatus[$count]="PASS"
+    nightlystatus[$count]="Pass"
     else
     echo "Failed"
-    nightlystatus[$count]="FAIL!"
+    nightlystatus[$count]="FAIL"
     fi
 
-    mv *.log $i
-    mv *.zip $i
+    mv *.log 60/$i
+    mv *.zip 60/$i
     ((count++))
     echo $count
 done
 
 for i in "${nightlystatus[@]}" 
 do
     echo $i
-    if [ $i = "PASS" ]
+    if [ $i = "Pass" ]
     then
-    buildStatus=0
+    buildStatus="Passed"
     echo "Test Passed!"
     else
-    buildStatus=1
+    buildStatus="Failed!"
     echo "Test failed, setting global test status to Failed!"
     break
     fi	
 done
 
-echo "Global Nightly Test Status $buildStatus"
+echo "Global Nightly Test Status-$buildStatus"
 
 drone exec --trusted -e test="sh tests/nightly/upload-logs.sh $input" -E nightly_test_secrets.yml --yaml .drone.nightly.yml
 
 rm nightly_mail.html
 
-if [ $buildStatus -eq 0 ]
-then
-echo "Success"
 cat <<EOT >> nightly_mail.html
 To: mwilliamson-staff-adl@vmware.com
 To: rashok@vmware.com
-Subject: VIC Nightly Run #$buildNumber
+Subject: VIC Engine Nightly $buildNumber-$buildStatus
```

Comment:
`Subject: VIC Engine Nightly for build $buildNumber - $buildStatus` would be more helpful.